src/data/faq_de.json

```diff
@@ -405,7 +405,8 @@
                         "anchor": "rooted_devices",
                         "active": true,
                         "textblock": [
-                            "Nein, die App wird auf gerooteten Geräten nicht unterstützt."
+                            "Ja, das ist möglich. Die App wird auf gerooteten Geräten jedoch nicht offiziell unterstützt.",
+                            "Zurzeit führen das Entsperren des Bootloaders or ähnliche Modifikationen (wie die Installation von microG) dazu, dass die Benutzung der optionalen Datenspende- und Befragungsfunktionen ohne Warnung fehlschlägt. De Grund dafür ist, dass diese Funktionen einen Integritätscheck über SafetyNet benötigen. Wir prüfen derzeit alternative Lösungen zur jetzigen Implementierung, siehe <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/356' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#356</a>."

```

Comment:
"or ähnliche" => "oder ähnliche"
"De Grund" => "Der Grund"

Reply:
thx all fixed

Reply:
@heinezen 
I'm not seeing the changes to 
"or ähnliche" => "oder ähnliche"
"De Grund" => "Der Grund"

Reply:
fixed